### ISSUE TITLE
[Accessibility]A11y_VSCode_ViewDebug_ScreenReader: Screenreader reads incorrect name for "Watch" toolbar.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52386]
**Environment Details:**
VSCode Version : 1.24.0
OS Version : HighSierra10.13.5

**Additional Details:**
MAS Violated : MAS 1.3.1
ScreenReader : VoiceOver

**Repro Steps:**
1. Launch VS Code while VoiceOver is ON.
2. Navigate to Activity Bar and select "View Debug" button.
3. Navigate to toolbars section under heading "Debug".

**Actual:**
Screenreader reads the incorrect name for "Watch" toolbar as "Expressions" toolbar.

**Expected:**
Screenreader should read proper name for "Watch" toolbar not as "Expressions" toolbar.

**Recommendations:**
Provide proper name to the toolbar
or,
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
The screenreader users will not be able to know the correct names for all the toolbars present on the screen.

**MAS Reference:**
MAS 1.3.1 - https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**Attachment For Reference:**
[A11y_VSCode_ViewDebug_VoiceOver_WatchToolbar.pptx](https://github.com/Microsoft/vscode/files/2117994/A11y_VSCode_ViewDebug_VoiceOver_WatchToolbar.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by isidorn** [comment]
Dup of https://github.com/Microsoft/vscode/issues/52307
